### ISSUE TITLE
Metric score

### ISSUE THREAD

**Issue by YINSONGHUI** [BaderLab/RETINA#2]
Dear author,

I can now sucessefully run train.py and inference2d scripts, but there's one thing related to the iou scroe i can't figure out:

1）the 2500 Training Iterations of Perez conflicts against your provided ckpt in "Tutorial model/epoch_2500.pth".Does the name "epoch_2500.pth" actually means "iteration_2500.pth"?

2）I used the pretrained ckpt "CEM500K_pretrained_(RETINA)/pretrain_200.pt" for finetuning the UroCell dataset, but the inference miou is higher than 0.610.

And for the Perez_lyso dataset ,i used the same "CEM500K_pretrained_(RETINA)/pretrain_200.pt" to finetune,but the inference miou is lower than 0.885.

**Does the reported socre include the background class?**

Can your provide the .yaml file for "Tutorial_quickstart/tutorial_quickstart.ipynb" , i want to find out whether i set wrong values resulting wrong iou score.

The code seems not to dealing with the selection of best ckpt in training,and only gives them identifiers of different epoch or iters.Maybe that's why i can't repeat your reported score?

**But  i can't get 0.885 score if i use the "Tutorial model/epoch_2500.pth" and set the data_dir to full "xxx\benchdata\perez\lyso\test"**
-I've tried this and got  0.8785755784964435

Thank you in advance for your support!
Best regards,
![Image](https://github.com/user-attachments/assets/d23122b0-c8f5-4b05-968f-56841ce2d7d4)

**Comment by CodeeXing** [comment]
Hi there, 
1. yes, this is typo, I have updated it.
2. what is the configuration you used? I am happy to here you even made RETINA a better performance.
3. Perez is binary segmentation so you could refer to the inference code I uploaded. There is a part related to this. And note that if you are testing the multiclass task (several classes), it would be different inference.
4. I tested it with the final iteration I listed, but you could try different ckpt iteration to select the best one. But based on my test, the final iteration I listed is optimal based on the convergence test shown in the paper.
5. The score of you test might be caused by the configuration set up or the normalization parameters you set up. You could play with them. And also for you own dataset, you could find good normalization parameters of your images. 
Best,
Cheng

**Comment by YINSONGHUI** [comment]
Hi,

Thank for your reply

> 1）the 2500 Training Iterations of Perez conflicts against your provided ckpt in "Tutorial model/epoch_2500.pth".Does the name "epoch_2500.pth" actually means "iteration_2500.pth"?
 
1.And i 'll try iteration1000 to inference again.
-later i tried this ,and got  0.49711367685105756. 
I guess it's due to the batch_size hyper_parameter which i set 8.

But after retraining using iteration_3000.pth ,i got higher miou=0.5289612617633056 again for class1-class6.
Here is the trained ckpt:
[guay_iter_3000](https://drive.google.com/file/d/1Xtp4uDDckS4CI1I7FsyNhDWHSoSlcsky/view?usp=drive_link)



2.I'll try to repeat the work based from the reported iterations,thanks.

3.I've used the normalization parameters form “[config_example.yaml](https://github.com/BaderLab/RETINA/blob/main/inference/config_example.yaml)” which is the only config file that you've provided,and it may result not the best score.

**Could you please upload all the config_yaml files coresponding to all the datasets?**

Thank for all your patience!

**Comment by CodeeXing** [comment]
Hello, 
Thanks for offering the guay checkpoint you trained and the IoU looks promising. I think it would be also helpful for other users with this. Could you upload it publicly without applying for access (I tried to access but it needed permission) so that others can use it as a reference as well.

In fact, we had already had all the configuration files publicly, as we have mentioned in the published paper and github , given by the reference of the 'CEM500K' paper. You could simply find all of them by clicking on the 'CEM500k' github directly.

Best,
Cheng

**Comment by YINSONGHUI** [comment]
Sorry i'm not familiar with the google drive, now the link should be avaliable to anyone.
And i put my tiraned ckpt of guay and urocell in a folder at this new [two_ckpt](https://drive.google.com/drive/folders/1qW_VwZfb4_jmBuIV7PgX_IStyvAPqcQI?usp=sharing)

I used all the test images to computer miou:753 for urocell,23 for guay , the normalizetion parameters both are   
mean: 0.5541282  std: 0.15759415 from the config_example.yaml file


I'll go find other config files at 'CEM500K' github,thanks!
